sprint-docs/02-demo-plan.md

```diff
@@ -0,0 +1,23 @@
+# Bookit Sprint #2: Demo (proposed)
+April 11, 2017
+
+We intend to demonstrate two user stories this week: Booking a room and Logging in.
+
+
+## Booking a room
+
+1) Open the Bookit web app. See the agenda view, set to today's date, showing meetings for all rooms.
+2) Click on an available space.
+3) See the room booking form appear on the left, pre-populated with the correct time and date.
+4) Add a meeting title.
```

Comment:
there must be a scenario branch with 'this time is not available'. Even two:
-local (UI)
-server (handled on server)